WcaOnRails/lib/database_dumper.rb

```diff
@@ -745,7 +745,10 @@ def self.development_dump(dump_filename)
           column_sanitizer == :copy ? "#{table_name}.#{column_name}" : "#{column_sanitizer} as #{column_name}"
         end.join(", ")
 
-        populate_table_sql = "INSERT INTO #{dump_db_name}.#{table_name} (#{column_sanitizers.keys.join(", ")}) SELECT #{column_expressions} FROM #{table_name} #{table_sanitizer[:where_clause]}"
+        # some column names like "rank" are reserved keywords starting mysql 8.0 and require quoting
```

Comment:
uber nit: We try to write our comments as full sentences, so starting with a capital letter and ending with a period

Reply:
Sure thing, conventions are important. I'll fix it.